location/location.js

```diff
@@ -25,7 +25,7 @@ var globals = require('can-globals/can-globals-instance');
  * @return {Object} The location object for this JavaScript environment.
  */
 globals.define('location', function(){
-	return globals.getKeyValue('global').location;
+	return globals.getKeyValue('location') || globals.getKeyValue('global').location;
```

Comment:
What’s the purpose of this change?

Reply:
@chasenlehara this allows ```globals.setKeyValue('location', location )``
i think this is more logic then ```globals.setKeyValue('global', { location: location })``` as this would also overwrite all other propertys on global. i thin can-globals should simply be seen as a global.